cookbooks/bcpc-hadoop/recipes/graphite_to_zabbix.rb

```diff
@@ -30,22 +30,26 @@
             :worker_count => node[:bcpc][:hadoop][:graphite][:worker_count]
            )
   mode 0744
-  owner "root"
-  group "root"
+  owner 'root'
+  group 'root'
 end
 
 template node[:bcpc][:hadoop][:zabbix][:query_graphite][:config_file] do
-  source "graphite.query_graphite.config.erb"
+  source 'graphite.query_graphite.config.erb'
   mode 0544
-  zabbix_triggers = node.run_state["zabbix_triggers"] or {}
-  variables(:queries => zabbix_triggers.map{ 
-    |host,item| item.select{|key, attr| !attr.key?('is_graphite_query') || attr['is_graphite_query'] }.map{ 
-      |key, attr| attr['query'] + " " + (attr['query_range'] ? attr['query_range'] : node[:bcpc][:hadoop][:graphite][:default_query_range])}
+  zabbix_triggers = node.run_state['zabbix_triggers'] or {}
+  variables(:queries => zabbix_triggers.map{
+    |host,item| item.select{|key, attr| !attr.key?('is_graphite_query') || attr['is_graphite_query'] }.map{
+      |key, attr| attr['query'] + ' ' + (attr['query_range'] ? attr['query_range'] : node[:bcpc][:hadoop][:graphite][:default_query_range])}
```

Comment:
It looks like you're rubocop/foodcriticing? If so `node[:bcpc][:hadoop][:graphite][:default_query_range]` should be `node['bcpc']['hadoop']...`